README.md

```diff
@@ -2,23 +2,39 @@
 
 Well hello there! 👋
 
-We want to assess your Front End proficiency and we've prepared this little test that should not take you a lot of time (ideally). It's a simple Newsletter subscription module.
+We want to assess your Front End proficiency and we've prepared this little test that should not take you a lot of time (ideally). The challenge is to create a simple newsletter aggregator application with a simple subscription form.
 
-We want to see your skills, specifically:
+Our goal is to identify your skill level in the following areas:
 
 * Front End concerns
 * Code semantics
 * Modularity
 * Attention to detail
 
-As a requirement, the module should look good on any device and work properly. There should be at least **one more element to select**. Doesn't have to be pixel perfect and, you're free to use placeholder text instead of what's on the design. 
+As a general requirement, the module should look good on any device and work properly. There should be at least **one more element to select**. Doesn't have to be pixel perfect and, you're free to use placeholder text instead of what's on the design. 
 
-To simplify the task, you can simulate that the form takes two seconds to submit and randomly notify that it has been submitted correctly, or that an error has occurred.    
+To simplify the task, you can simulate that the form takes two seconds to submit and randomly notify that it has been submitted correctly, or that an error has occurred.
 
-Use either plain JavaScript (no TypeScript) or React. The usage of any other library is **completely discouraged** since there should be no need these days. If you're taking the React route, please `useHooks` as that's what we would like to assess. It is fine though (and even encouraged) to use [Create React App](https://github.com/facebook/create-react-app#creating-an-app) if you're taking the React path. 
+## Completion Instructions:
 
-![Composition](/assets/composition.png)
+- Initialize a package using either `yarn` or `npm`.
+- Install any required pacakges and prepare the code and folder structure however you like.
+- _IF_ you are using ES6 or similar code that must be transpiled, add a `build`, `start` or `dev` script to the `package.json`
+- Build the form using your chosen code paradigm. You may use ES5/6, React, or Vanilla Javascript, etc. Any other libraries, or syntax supersets (ie. Typescript) are **completely discouraged**.
```

Comment:
```suggestion
- Build the form using your chosen code paradigm. You may use ES5/6, React, or Vanilla Javascript, etc.
```

I'd actually remove that part. If someone opts to use TS or another library they should be ready to explain the choice. We might learn something from it too.